llvm/lib/Target/RISCV/GISel/RISCVRegisterBankInfo.cpp

```diff
@@ -478,7 +478,6 @@ RISCVRegisterBankInfo::getInstrMapping(const MachineInstr &MI) const {
     LLT Ty = MRI.getType(MI.getOperand(2).getReg());
 
     unsigned Size = Ty.getSizeInBits();
-    assert((Size == 32 || Size == 64) && "Unsupported size for G_FCMP");
```

Comment:
I deleted this assert since getFPValueMapping has the same assert and is called a couple lines down.